src/aria/tabs/tabs.ts

```diff
@@ -202,13 +188,25 @@ export class TabList implements OnInit, OnDestroy {
   private _hasFocused = signal(false);
 
   constructor() {
-    afterRenderEffect(() => this.selectedTab.set(this._selection()[0]));
-
     afterRenderEffect(() => {
       if (!this._hasFocused()) {
         this._pattern.setDefaultState();
       }
     });
+
+    afterRenderEffect(() => {
+      const tab = this._pattern.selectedTab();
+      if (tab) {
+        this.selectedTab.set(tab.value());
+      }
+    });
+
+    afterRenderEffect(() => {
+      const value = this.selectedTab();
+      if (value) {
+        this._pattern.open(value);
+      }
+    });
```

Comment:
Make the individual tabs the source of truth for this. We can still expose a readonly `selectedTab` at the top level